nixos/modules/services/networking/firefox/sync-server.nix

```diff
@@ -124,7 +124,13 @@ in
       description = "Firefox Sync Server";
       wantedBy = [ "multi-user.target" ];
       path = [ pkgs.pythonPackages.pasteScript pkgs.coreutils ];
-      environment.PYTHONPATH = "${pkgs.pythonPackages.syncserver}/lib/${pkgs.pythonPackages.python.libPrefix}/site-packages";
```

Comment:
I don't know what you mean by that. How would I do it ? I'm not very familiar with python+nix